diff_drive_controller/package.xml

```diff
@@ -14,12 +14,16 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <build_depend>cmake_modules</build_depend>
   <build_depend>controller_interface</build_depend>
   <build_depend>nav_msgs</build_depend>
   <build_depend>realtime_tools</build_depend>
   <build_depend>tf</build_depend>
   <build_depend>urdf</build_depend>
+  <build_depend>eigen</build_depend>
```

Comment:
@mikepurvis Do you know if this is actually needed when cmake_modules is already a build depend, following [this](https://github.com/ros/cmake_modules/blob/0.3-devel/README.md#usage)?